etc/deploy/tracing/README.md

```diff
@@ -0,0 +1,57 @@
+# Overview
+
+Pachyderm has the ability to trace requests using Jaeger (an implementation of
+the OpenTracing API written by Uber). This can be useful when diagnosing slow
+clusters.
+
+TODO add screenshot
+
+# Quickstart
+
+## Collecting Traces
+
+To use tracing in Pachyderm, you need to:
+1. Run Jaeger in Kubernetes
+
+    ```
+    kubectl apply -f etc/deploy/tracing/jaeger-all-in-one.yaml
+    ```
+
+2. Point Pachyderm at Jaeger
+
+    ```
+    # For pachctl
+    $ export JAEGER_ENDPOINT=localhost:14268
+    $ jaeger_pod="$(kubectl get po -l app=jaeger -o jsonpath='{.items[].metadata.name}')"
+    $ kubectl port-forward "po/${jaeger_pod}" 14268 &  # Collector port
```

Comment:
Kubernetes has recently added the ability to port-forward to a service, which I think basically does the work you're doing here for you.

Reply:
:+1: Updated this section